### DISCUSSION TITLE
The ability to switch modes for the player using the Editor API

### DISCUSSION THREAD

**Discussion by xKingDark** [Mojang/minecraft-editor#114] [Feedback]
It would be handy to have the ability to switch modes for the player using the Editor API

**Comment by conmaster2112** [comment]
Do you mean gamemodes right?

**Reply by xKingDark** [reply]
No, the Ability to switch from Crosshair mode to Tool mode

**Comment by minecraftchloe** [comment]
This is a great idea! Can you tell us more about the scenarios you're thinking of with this? It'll help us with interpretation and shaping an implementation proposal. Thanks! :)

**Reply by xKingDark** [reply]
I'm currently trying to create a cinematic camera extension for Editor mode and having the ability to switch the player into Crosshair mode when playing then switch back into Tool mode would be cool.
Also knowing which mode is selected would be useful too
